lib/nerves_hub_web/views/org_user_view.ex

```diff
@@ -1,10 +1,10 @@
 defmodule NervesHubWeb.OrgUserView do
   use NervesHubWeb, :view
 
-  alias NervesHub.Accounts.User
-
   def role_options() do
-    User.Role.__enum_map__() |> Enum.map(fn opt -> {format_option(opt), opt} end)
+    Enum.map([:admin, :delete, :write, :read], fn opt ->
```

Comment:
```suggestion
  
Ecto.Enum.values(OrgUser, :role)
|> Enum.map(fn opt ->
```
I'm pretty sure you can use the defined enum values like this (or close to this) rather than hardcoding

Reply:
You can! This is much better